src/page-racial-disparities/PageRacialDisparities.js

```diff
@@ -0,0 +1,543 @@
+            Sentences are imposed based on the type and severity of crime. In
+            many cases, courts have some discretion over what sentence to impose
+            on a person convicted of an offense. In the aggregate, these
+            variations in sentencing add up to significant trends.
+          </p>
+          <p>
+            In the last 6 months,{" "}
+            <DynamicText>
+              {formatAsPct(metrics.proportionIncarcerated)}
+            </DynamicText>{" "}
+            of <DynamicText>{ethnonym}</DynamicText> under DOCR jurisdiction
+            were incarcerated while{" "}
+            <DynamicText>
+              {formatAsPct(metrics.proportionSupervision)}
+            </DynamicText>{" "}
+            were on supervision, a{" "}
+            <DynamicText>
+              {metrics.proportionIncarcerated >
+              metrics.proportionIncarceratedOverall
+                ? "greater"
+                : "smaller"}
+            </DynamicText>{" "}
+            percent incarcerated than the overall distribution of{" "}
+            <DynamicText>
+              {formatAsPct(metrics.proportionIncarceratedOverall)}
+            </DynamicText>{" "}
+            incarcerated and{" "}
+            <DynamicText>
+              {formatAsPct(metrics.proportionSupervisionOverall)}
+            </DynamicText>{" "}
+            on supervision.
+          </p>
+        </>
+      ),
+    },
+    {
+      title: "How can parole grant rates impact disparities?",
+      description: (
+        <>
+          <p>
+            People sentenced to a prison term can serve the end-portion of their
+            term while supervised in the community, through the parole process.
+          </p>
+          <p>
+            The parole process is governed by the Parole Board, an independent
+            commission that works closely with the DOCR. In 2019, under guidance
+            from Governor Burgum and then-Director of Corrections Leann Bertsch,
+            The DOCR and the Parole Board began tracking and reporting racial
+            data for the parole process in order to monitor and reduce
+            disparities in the population granted parole.
+          </p>
+          <p>
+            In the last 6 months, <DynamicText>{ethnonym}</DynamicText>{" "}
+            comprised{" "}
+            <DynamicText>{formatAsPct(metrics.paroleRate)}</DynamicText> of the
+            individuals released on parole. They make up{" "}
+            <DynamicText>
+              {formatAsPct(metrics.prisonPopulationRate)}
+            </DynamicText>{" "}
+            of the overall prison population.
+          </p>
+        </>
+      ),
+    },
+    {
+      title: "How can community supervision impact disparities?",
+      otherControls: (
+        <Dropdown
+          label="Supervision Type"
+          onChange={setSupervisionType}
+          selectedId={supervisionType}
+          options={supervisionTypeOptions}
+        />
```

Comment:
In this dropdown, when I select "Everyone" or "Parole" the dropdown stays fixed to the right side of the section header.  However, when I select "Probation", the control wraps to the next line.  See the screenshots below:

**Normal**

<img width="1034" alt="Screen Shot 2020-07-31 at 10 42 21 AM" src="https://user-images.githubusercontent.com/25503/89046330-cd962100-d31a-11ea-8471-2b77870541cd.png">

**Wrapped Control**

<img width="1023" alt="Screen Shot 2020-07-31 at 10 42 31 AM" src="https://user-images.githubusercontent.com/25503/89046347-d555c580-d31a-11ea-82d9-d9a02e1b2257.png">

Reply:
oh yeah I noticed this too ... planning to fix it as part of #86 because I think some of the layout structure will change a bit in a way that will affect this